Resources/doc/reference/configuration/tags.rst

```diff
@@ -29,6 +29,25 @@ incompatible proxies:
 
 Custom HTTP header that tags are stored in.
 
+.. include:: ../../includes/expression-language.rst
+
+Your custom expression functions can then be used in both the ``tag_expressions``
+section of the tag configuration and ``@tag`` :ref:`annotations<tag>`.
+
+.. code-block:: yaml
+
+    # app/config/config.yml
+    services:
+        acme.expression_language:
+            class: Acme\TestBundle\Component\ExpressionLanguage
```

Comment:
should we make the code example an include as well? or actually, i would just say that you define a custom service without actually showing how to define the service, its a bit much details